assets/js/components/Board/index.js

```diff
@@ -1,46 +1,109 @@
 import React, { Component } from 'react';
 import Square from '../Square'
+import Message from '../Message'
 import axios from 'axios'
 
 class Board extends Component {
     constructor(props) {
         super(props)
         this.state = {
-            moves: Array(9)
+            gameStatus: "",
+            moves: [],
+            currentPlayer: "",
+            message: "Player one,",
+            error: ""
         }
     }
 
-    selectSquare(position_number) {
+    componentDidMount() {
+        axios.get('/newgame')
```

Comment:
Is it really the responsibility of the `Board` class to make the API call?

Reply:
I moved the call over to the game component.